new-empty-theme.php

```diff
@@ -0,0 +1,169 @@
+<?php
+/**
+ * Generates a theme based on the empty theme.
+ * Simplified version of https://github.com/Automattic/underscores.me/blob/master/plugins/underscoresme-generator/underscoresme-generator.php
+ */
+
+class Generate_Theme {
+
+	protected $theme;
+
+	function __construct() {
+		echo "What is the name of your theme?: ";
+		$handle = fopen ("php://stdin","r");
+
+		$this->theme['name'] = trim(fgets($handle));
+		$this->theme['slug'] = $this->sanitize_title_with_dashes($this->theme['name']);
+		$this->theme['functions_slug'] = str_replace( '-', '_', $this->theme['slug'] );
+		$this->create_zip();
+	}
+
+	function create_zip() {
+
+		if($this->theme['name'] === ''){
+			echo "ABORTING!\n";
+			exit;
+		}
+
+		$zip = new ZipArchive;
+		$zip_filename = sprintf( '%s.zip', md5( print_r( $this->theme['name'], true ) ) );
```

Comment:
This results in a zip with a name as an md5 hash e.g. `098f6bcd4621d373cade4e832627b4f6.zip`. Is this intentional?

Reply:
I think the slug would be better suited for the theme name, e.g. `my-theme.zip` instead of the MD5.

Reply:
yeah, it's the approach by the original underscores' script but we can change this definitely